shared/src/main/scala/async/CompletionGroup.scala

```diff
@@ -43,11 +43,10 @@ class CompletionGroup extends Cancellable.Tracking:
   def isCancelled = canceled
 
 object CompletionGroup:
-
   /** A sentinel group of cancellables that are in fact not linked to any real group. `cancel`, `add`, and `drop` do
     * nothing when called on this group.
     */
-  object Unlinked extends CompletionGroup:
+  val Unlinked = new CompletionGroup:
```

Comment:
When initializing `Unlinked`, we initialize `CompletionGroup` -> `Cancellable.Tracking` -> `Cancellable`, which assigns the default linked group to `CompletionGroup.Unlinked`. This is a bug that causes Scala.js to infinite loop. This is just a patch, needs to properly handle before we merge.